webapp/graphite/functions/params.py

```diff
@@ -85,7 +86,24 @@ def validateInterval(value):
 
 
 def validateSeriesList(value):
-    return list(value)
+    if not isinstance(value, list):
+        raise ValueError('Invalid value type, it is not a list: {value}'.format(value=repr(value)))
+
+    for serie in value:
```

Comment:
```suggestion
    for series in value:
```
series is already singular.

Reply:
that's a bug in English 😂